po/R-zh_CN.po

```diff
@@ -416,18 +478,33 @@ msgid ""
 msgstr ""
 "但i是一个 data.table (或者是字符向量)，必须使用 'on=' 参数指明参与连接的列 "
 "(参见 ?data.table)，可以是keying x(比如，已排序过，和标记已排序过，请参见?"
-"setkey)，或者是在x和i共用列的名字(比如，自然连接)。如果x有在内存被排序过，"
-"Keyed连接的速度会在非常大的数据上有较明显的提高。"
+"setkey)，或者是在x和i共用列的名字(比如，自然连接)。如果x有在内存被排序过，键"
+"(keyed)连接的速度会在非常大的数据上有较明显的提高。"
```

Comment:
Where did the original message go?

Reply:
It's just above (you can click the blue arrow), just got truncated by the GH preview